### ISSUE TITLE
Plumbing review/markups - 9/9/2018

### ISSUE THREAD

**Issue by theoryshaw** [OpeningDesign/LaundryMat#3]
Hi, 
@rvzenteno, 
http://sketch.openingdesign.com/d/Nnts7PxdcC

from [this](https://github.com/OpeningDesign/LaundryMat/commit/6f96eaffffa4ea62739c278ec03ac86cf24e5d05) commit

@apbock, feel free to markup and/or modify as necessary. 

Thanks, Ryan

**Comment by apbock** [comment]
Can't view the entire sketch on my phone, but I do see a question about the washer supplies. The 1-1/2 mains should be adequately sized for the multiple units (I will confirm when I do a full review) and separate branches can take off from these to serve each unit. Wherever we have a unit with detergent feed in these groups, we'll need to put a backflow preventer on the mains before they branch off. The backflow preventer needs to be visible and accessible. 

I'll have a closer look at this when I'm back on my laptop.

**Comment by theoryshaw** [comment]
Hi @rvzenteno, 
If we can, let's finish this up for permit issuance this week. 
The client has been asking to wrap this up. 

Thanks Guys, Ryan

**Comment by rvzenteno** [comment]
Hello @theoryshaw,

I was waiting on the comments from Aaron full review to continue. As soon as I get that I'll proceed. 

Thank you,

**Comment by apbock** [comment]
Sorry, didn’t realize you were waiting for me. You will need individual sets of supplies for each unit as you’ve eluded to. You will also need a trench drain with sediment bucket to satisfy the code requirements. Also add a backflow preventer for the MFR units as discussed before. The other notes you have will work. Please post the final set/model when you have it. Thanks.

**Comment by theoryshaw** [comment]
Thanks Aaron!

**Comment by rvzenteno** [comment]
Hello @theoryshaw @apbock I've update the model with all the comments posted to github.

**Comment by apbock** [comment]
@theoryshaw @rvzenteno  I’ve posted my markups in PDFs in the BIM folder. I also uploaded a model with a backflow preventer detail in it for the MFR washers, but that’s the only change I made to the actual model; the rest of the revisions should be taken from the marked up PDFs. Some of the revisions come from code requirements, others from what I think is good design, and another to see if we can take advantage of existing vent piping. But overall there is not much more to do.

If you recall from the vacation home, we filled out a hydraulic calculation worksheet for permit. I don’t think they will require that here since it is a remodel, but I can fill it out quickly if they do; I would just need a pressure reading taken from somewhere in the existing space.

@rvzenteno, if you have questions about any of my markups, please post them here. I will be monitoring closely until @theoryshaw says we’re good to go. Thanks.

**Comment by theoryshaw** [comment]
Thanks Aaron!

Fyi... [Here's](https://www.dropbox.com/s/ievz3ljm9lm1kwh/2018-09-15_07-59-36.mp4?dl=0) a quick video on making intelligent tags for the washers.

If we can, let's shoot for sending the final plans out monday morning.

Thanks Guys!

**Comment by rvzenteno** [comment]
@apbock @theoryshaw, Modifications done. PDF files created.

Thank you,

**Comment by theoryshaw** [comment]
Thanks Roberto, 

Have some markups here: 
`Transfer/20180916 - from Ryan - markups/`

@apbock, can you also check these markups, and correct anything you see?

Would be great to finish this up tonight if possible.

Please put anything you transfer in the `transfer` folder

Thanks guys, Ryan

**Comment by apbock** [comment]
@theoryshaw I've added my comments to your markups. They are in the same folder.


@rvzenteno Once those are incorporated we'll be complete. Thanks for the quick turnaround.

**Comment by theoryshaw** [comment]
Thanks Aaron!

If you want to keep those sections, but not print them... 

![image](https://user-images.githubusercontent.com/507113/45599667-4dc86d00-b9b5-11e8-8f9d-8f54be9745c3.png)

![image](https://user-images.githubusercontent.com/507113/45599669-5de04c80-b9b5-11e8-963c-401812a6f685.png)

**Comment by rvzenteno** [comment]
@theoryshaw @apbock , Guys, Files have been revised and uploaded.

**Comment by theoryshaw** [comment]
Don't we sill need a sediment bucket?

**Comment by rvzenteno** [comment]
@theoryshaw Aaron had this on his markups, that is why I eliminated it. Did I misunderstand something?

![image](https://user-images.githubusercontent.com/600617/45622346-ae0cec80-ba51-11e8-98bb-10c52b7c4ef7.png)

**Comment by theoryshaw** [comment]
Sounds great thanks

**Comment by apbock** [comment]
Yes, code allows the strainer in the trench drain to act as a solids-catcher rather than have a dedicated sediment bucket. I elected to not put a separate unit out in the walkways.
